README.md

```diff
@@ -195,6 +196,55 @@ For example:
 The available keywords for the keys of a period object are: `year`, `quarter`,
 `month`, `week`, `day`, `hour`, `minute`, `second`, and `millisecond`.
 
+### `dedupe`
+
+Removes duplicates from an array. Only the first item that is a match will be
+kept. Any value that is `undefined` will not count as unique and will be removed.
+
+For example:
+
+```javascript
+const data = [undefined, 'first', 'first', 'second', 0, 1, 2, 2, null]
+```
+
+will return:
+
+```javascript
+const result = ['first', 'second', 0, 1, 2, null]
+```
+
+Dedupe can recieve a path property which removes duplicates based on an embedded
+values. If no path is specified, the first layer of the array will be used.
```

Comment:
I don't really understand this sentence ... 🤔

Reply:
Don't think it's necessary actually.  You can just remove it.  Comes from the way I was trying to document dedupe before.